frontend/src/core/components/AuthSidebar/AuthSidebar.scss

```diff
@@ -2,18 +2,37 @@
   margin-top: 35%;
 
   h1 {
-    color: #ffffff;
+    width: 254px;
+    height: 90px;
+    font-family: OpenSans;
```

Comment:
About this properties like font we have a scss file, that has font properties, please import that file(Variables/Font.scss) and use it here.